demos/TicTacToe/Classes/main_menu_scene.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "main_menu_scene.h"
 
+#include <cocos\ui\UITextField.h>
```

Comment:
Check the order of these includes.  I think this one should use quotes, and all quoted headers should be together.  System headers in <> (ie: regex) should appear first.   Checkout the [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).

Reply:
This include should be in ""s, as its not a system library.